.gitignore

```diff
@@ -83,6 +83,7 @@ instance/
 
 # Sphinx documentation
 docs/_build/
+docs/autoapi/
```

Comment:
We should rename the autoapi folder to _autoapi

Reply:
Done.